server/app/views/questiontypes/AddressQuestionFragment.html

```diff
@@ -0,0 +1,137 @@
+    ></div>
+
+    <!-- TODO(#7007): Use ErrorDisplayMode to determine whether to show field errors. -->
+    <!-- Street address -->
+    <th:block th:with="inputId=${#strings.randomAlphanumeric(8)}">
+      <label
+        class="usa-label"
+        th:for="${inputId}"
+        th:text="#{label.street}"
+      ></label>
+      <input
+        class="usa-input"
+        th:id="${inputId}"
+        th:name="${addressQuestion.getStreetPath()}"
+        th:value="${addressQuestion.getStreetValue().orElse('')}"
+        th:aria-invalid="${!addressQuestion.getValidationErrors().isEmpty()}"
+        th:aria-required="${!question.isOptional()}"
+        autocomplete="address-line1"
+      />
+      <div
+        th:replace="~{questiontypes/QuestionBaseFragment :: validationErrors(${addressQuestion.getValidationErrors().get(addressQuestion.getStreetPath())})}"
+      ></div>
+    </th:block>
+
+    <!-- Street address line 2 -->
+    <th:block th:with="inputId=${#strings.randomAlphanumeric(8)}">
+      <label
+        class="usa-label"
+        th:for="${inputId}"
+        th:text="#{label.addressLine2}"
+      ></label>
+      <input
+        class="usa-input"
+        th:id="${inputId}"
+        th:name="${addressQuestion.getLine2Path()}"
+        th:value="${addressQuestion.getLine2Value().orElse('')}"
+        th:aria-invalid="${!addressQuestion.getValidationErrors().isEmpty()}"
+        autocomplete="address-line2"
+      />
+      <div
+        th:replace="~{questiontypes/QuestionBaseFragment :: validationErrors(${addressQuestion.getValidationErrors().get(addressQuestion.getLine2Path())})}"
+      ></div>
+    </th:block>
+
+    <!-- City -->
+    <th:block th:with="inputId=${#strings.randomAlphanumeric(8)}">
+      <label
+        class="usa-label"
+        th:for="${inputId}"
+        th:text="#{label.city}"
+      ></label>
+      <input
+        class="usa-input"
+        th:id="${inputId}"
+        th:name="${addressQuestion.getCityPath()}"
+        th:value="${addressQuestion.getCityValue().orElse('')}"
+        th:aria-invalid="${!addressQuestion.getValidationErrors().isEmpty()}"
+        th:aria-required="${!question.isOptional()}"
+        autocomplete="address-level2"
+      />
+      <div
+        th:replace="~{questiontypes/QuestionBaseFragment :: validationErrors(${addressQuestion.getValidationErrors().get(addressQuestion.getCityPath())})}"
+      ></div>
+    </th:block>
+
+    <!-- State -->
+    <th:block th:with="inputId=${#strings.randomAlphanumeric(8)}">
+      <label
+        class="usa-label"
+        th:for="${inputId}"
+        th:text="#{label.state}"
+      ></label>
+      <select
+        class="usa-select"
+        th:id="${inputId}"
+        th:name="${addressQuestion.getStatePath()}"
+        th:aria-invalid="${!addressQuestion.getValidationErrors().isEmpty()}"
+        th:aria-required="${!question.isOptional()}"
```

Comment:
```suggestion
        th:aria-required="${!question.isOptional()}"
        autocomplete="address-level1"
```

Looks like `autocomplete` is a valid attribute for select elements. Can you include `address-level1`? If you purposefully left it out maybe leave a comment on why.